ferrilata-devilline.IntegrationTests/Scenarios/ApiControllers/Badges/ApiPostLevelByBadgeId.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios.ApiControllers.Badges
+{
+    class ApiPostLevelByBadgeId
```

Comment:
Why is this empty private class necessary?

Reply:
deleted